Ginger/GingerCoreNET/ActionsLib/General/ActSetVariableValue.cs

```diff
@@ -274,7 +274,11 @@ public override void Execute()
                 Error = "Unknown set " + GingerDicser.GetTermResValue(eTermResKey.Variable) + " value operation.";
                 return;
             }
-
+            if (Var.Value.Contains("Value is at the last in the list"))
```

Comment:
not sure i like this approach with setting the value with the error + probably will cause issue when call auto Value from other places- let's talk on it on Sunday